ampersand-search-select.js

```diff
@@ -0,0 +1,241 @@
+;(function() {
+  var d3 = require('d3');
+  var _ = require('lodash');
+
+  var AmpersandState = require('ampersand-state');
+  var AmpersandView = require('ampersand-view');
+  var AmpersandSubCollection = require('ampersand-subcollection');
+
+  var SearchSelectState = AmpersandState.extend({
+    props: {
+      selected: 'object'
+    },
+    session: {
+      query: [ 'string', false, ''],
+      data: 'object',
+      idAttribute: 'string',
+      imageAttribute: 'string',
+      queryAttribute: 'string',
+
+      // Private Variables
+      _data: 'object',
```

Comment:
Sticking to the idea that `.toJSON()` should only give you the stuff you should care about. I feel like the other thing you might care about is the `query`, but everything else besides the private variables are things the user sets.